src/phylum/ci/ci_base.py

```diff
@@ -349,6 +368,8 @@ def analyze(self, analysis: dict) -> ReturnCode:
             returncode = ReturnCode.SUCCESS
             output = SUCCESS_COMMENT
 
+        project_id = analysis.get("project", "00000000-0000-0000-0000-000000000000")
```

Comment:
An odd default... But I guess it's better than raising an exception since results with a bad link are better than no results at all.

Reply:
Yeah...that was the thought.